submissions/Oleksii_Drohachov/Tiny_JS_world/index.js

```diff
@@ -0,0 +1,101 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+catsPhrase = "meow-meow skinbag";
+
+const tinyWorldInhabitants = [
+  {
+    species: "dog",
+    name: "Barbos",
+    gender: "male",
+    legs: 4,
+    hands: 0,
+    saying: "gav-gav kaje pes",
+    friends: ["definitely everybody in the world !"],
+  },
+  {
+    species: "cat",
+    name: "Sonya",
+    gender: "female",
+    legs: 4,
+    hands: 0,
+    saying: catsPhrase,
+    friends: ["definitely nobody except her own slaves"],
+  },
+  {
+    species: "human",
+    name: "Oleksii",
+    gender: "male",
+    legs: 2,
+    hands: 2,
+    saying: "Wanna be frontend ninja in future!",
+    friends: ["Alex", "Victoria", "every lovely kottan on the course"],
+  },
+  {
+    species: "human",
+    name: "Victoria",
+    gender: "female",
+    legs: 2,
+    hands: 2,
+    saying: "When will you bring the money to home, honey?",
+    friends: ["Natasha", "Katya", "Lada"],
+  },
+  {
+    species: "catWoman",
+    name: "Anjela",
+    gender: "female",
+    legs: 2,
+    hands: 2,
+    saying: catsPhrase,
```

Comment:
This approach won't ensure catWoman's saying when cat's will change.
You do not need to fix this here. But you will need different approach when you are back to this JS World after the next task.